kubernetes/services/dpm_manager.yaml

```diff
@@ -60,6 +66,7 @@ data:
     microservices.netwconfigmanager.service.url=netwconfigmanager-service
     microservices.zeta.management.url=http://10.213.43.90
     microservices.gateway.service.url=http://gatewaymanager-service.default.svc.cluster.local:9015
+    microservices.connectTimeout=5
```

Comment:
The unit is inconsistent throughout the PR. 

The unit is minute in the yaml file and in code implementation, while is second in application.properties.

Reply:
This is for grpc connections (DPM TO NCM).